### DISCUSSION TITLE
Calibration failed, device errors: 0x0

### DISCUSSION THREAD

**Discussion by StevenCellist** [jgromes/RadioLib#890] [Q&A]
I have multiple Wireless Trackers, and one of them unexpectedly throws me an error:
Calibration failed, device errors: 0x0
The result is a state = -2.
I found only one issue from 2019 mentioning TXCO, but also a subsequent commit that seems to solve it, so I don't really know what to do. Any suggestion? Cheers!

**Comment by StevenCellist** [comment]
Huh, while typing this, it actually came alive at some point. Lesson learned.. be a bit more patient :)

**Reply by jgromes** [reply]
The error is a bit weird, typically some device errors would be reported when calibration fails. Let me know if you find a reliable way to reproduce it.

**Reply by StevenCellist** [reply]
It always happens first time after power cycle; it doesn't happen after deepsleep resume. Currently ignoring the behaviour, will try to follow up later.
